dvc/command/lock.py

```diff
@@ -22,29 +24,34 @@ def run(self, unlock=False):
 
 
 class CmdLock(CmdLockBase):
-    def run(self):
+    def run(self, _=False):
```

Comment:
Not sure what this is for.

Reply:
Same in run() below.

Reply:
something like this - https://help.semmle.com/wiki/display/PYTHON/Signature+mismatch+in+overriding+method

Reply:
@shcheklein Not a big fan of such things. Could we at least put a comment about it? Right now it looks like a left-over. Or maybe even rename CmdLockBase.run() into CmdLockBase._run() so we don't even have to bother with this stuff.

Reply:
Not big fan either, don't like potentially breaking the interface though. Let me take a look if there are other options and what ppl do about it. What do you mean by renaming? I'm not sure I understand you suggestion with renaming.

Reply:
@shcheklein So CmdLock inherits from CmdLockBase, which has a method called run() with two args. This is where that pycharm warning comes from. If we just rename it to _run() and use self._run() in CmdLock/Unlock, then that warning should disappear.